File: C#/JOIN_CLASSES/JOIN_CLASSES/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JOIN_CLASSES
{
    class Pessoa
    {
        public string Nome { get; set; }
        public double CPF { get; set; }
        public Endereco End;

        public Pessoa()
        {
            Nome = "";
            CPF = 0;
            End = new Endereco();
        }
        public Pessoa(string xNome, double xCPF)
        {
            Nome = xNome;
            CPF = xCPF;
            End = new Endereco();
        }
        public void addEnd(string xrua,string xcep)
        {
            End.rua = xrua;
            End.cep = xcep;
        }
        /*public void addEnd(string xcep,string xrua)
        //cep foi convertido de double para string(metodo foi inutilizado)
        {
            End.rua = xrua;
            End.cep = xcep;
        }*/
        public void addEnd(Endereco xend)
        {
            End = xend;
        }
        public Endereco retEnd()
        {
            return End;
        }
    }
}

File: Java/Aprendendo_20_05_2018/src/Pessoa.java

public class Pessoa {

	public Pessoa() {
		Endereco end = new Endereco();
	}
	
	private String nome;
	private double salario;
	private int cpf;
	
	public String Get_Nome(){
		return nome;
	}
	public void Set_Nome(String xNome) {
		nome = xNome;
		
	}
	public double Get_Salario() {
		return salario;
	}
	public void Set_Salario(double xSalario) {
		salario = xSalario;
		
	}
	public int Get_CPF() {
		return cpf;
	}
	public void Set_CPF(int xCPF) {
		cpf = xCPF;
	}

}
